aspnetcore/fundamentals/index.md

```diff
@@ -16,97 +16,121 @@ ms.custom: H1Hack27Feb2017
+* [Authentication](xref:security/authentication/index)
 
 You can use any [OWIN](http://owin.org)-based middleware with ASP.NET Core, and you can write your own custom middleware.
 
-For more information, see [Middleware](middleware.md) and [Open Web Interface for .NET (OWIN)](owin.md).
+For more information, see [Middleware](xref:fundamentals/middleware) and [Open Web Interface for .NET (OWIN)](xref:fundamentals/owin).
 
 ## Servers
 
-The ASP.NET Core hosting model does not directly listen for requests; rather it relies on an HTTP server implementation to forward the request to the application. The forwarded request is wrapped as a set of feature interfaces that the application then composes into an `HttpContext`.  ASP.NET Core includes a managed cross-platform web server, called [Kestrel](servers/kestrel.md) that you would typically run behind a production web server like [IIS](https://iis.net) or [nginx](http://nginx.org).
+The ASP.NET Core hosting model does not directly listen for requests; rather, it relies on an HTTP server implementation to forward the request to the application. The forwarded request is wrapped as a set of feature interfaces that the application then composes into an `HttpContext`. ASP.NET Core includes a managed, cross-platform web server, called [Kestrel](xref:fundamentals/servers/kestrel), that you would typically run behind a production web server like [IIS](https://iis.net) or [nginx](http://nginx.org).
 
-For more information, see [Servers](servers/index.md) and [Hosting](hosting.md).
+For more information, see [Servers](xref:fundamentals/servers/index) and [Hosting](xref:fundamentals/hosting).
 
 ## Content root
 
-The content root is the base path to any content used by the app, such as its views and web content. By default the content root is the same as application base path for the executable hosting the app; an alternative location can be specified with *WebHostBuilder*.
+The content root is the base path to any content used by the application, such as its views and web content. By default, the content root is the same as application base path for the executable hosting the application; an alternative location can be specified with `WebHostBuilder`.
 
 ## Web root
 
-The web root of your app is the directory in your project for public, static resources like css, js, and image files. The static files middleware will only serve files from the web root directory (and sub-directories) by default. The web root path defaults to *<content root>/wwwroot*, but you can specify a different location using the *WebHostBuilder*.
+The web root of your application is the directory in your project for public, static resources like CSS, JavaScript, and image files. By default, the static files middleware will only serve files from the web root directory and its sub-directories. The web root path defaults to *<content root>/wwwroot*, but you can specify a different location using the `WebHostBuilder`.
```

Comment:
The web root of an application is the directory in the project containing public, static resources like CSS, JavaScript, and image files. By default, the static files middleware will only serve files from the web root directory and its sub-directories. See [ working with static files}(xref:fundamentals/static-files) for more info. The web root path defaults to *<content root>/wwwroot*, but you can specify a different location using the `WebHostBuilder`.

do you need to escape or code fence <?
When reworking steves articles I try to replace a bunch of application with app and remove as much compiler speak as possible.